### PR TITLE
[software_renderer] Fix build.

### PR DESCRIPTION
**Pull request by mpimenov**
Missing std includes resulted in build problems on some systems
as reported here: https://github.com/mapsme/omim/issues/11976

### CODE REVIEW

**Comment by mpimenov** [comment]
JTALL

File: software_renderer/rect.h [hunk 1]
```diff
@@ -3,8 +3,10 @@
 
 #include "software_renderer/point.h"
 
+#include <algorithm>
```
Comment (tatiana-yan) [inline software_renderer/rect.h:6]: std::min далее?

Comment (tatiana-yan) [inline software_renderer/rect.h:6]: не нужно include cmath и s/fabs/std::fabs далее?

Comment (mpimenov) [reply software_renderer/rect.h:6]: done

**Comment by mpimenov** [comment]
JTALL

**Pull request merged**